packages/react-core/src/components/Slider/_tests_/Slider.test.tsx

```diff
@@ -79,3 +79,13 @@ describe('slider', () => {
     expect(asFragment()).toMatchSnapshot();
   });
 });
+
+test('renders slider with aria-labelledby', () => {
+  const { asFragment } = render(<Slider value={50} aria-labelledby="slider aria-labelledby" />);
+  expect(asFragment()).toMatchSnapshot();
+});
+
+test('renders slider with aria-describedby', () => {
+  const { asFragment } = render(<Slider value={50} aria-describedby="slider aria-describedby" />);
+  expect(asFragment()).toMatchSnapshot();
+});
```

Comment:
@wise-king-sullyman do you think instead of snapshots these tests could instead check for `toHaveAccessibleName` or `toHaveAttribute`?

Reply:
@thatblindgeye I *believe* so, another approach could probably be to use the `ByLabelText` selector. 

I'm not positive in the case of aria-describedby what the best route would be. I'm going to do some testing/investigation on that topic and report back shortly.

Reply:
I think the first test would be more true to the way this functionality would be used by refactoring to something like:

```js
test('renders slider with aria-labelledby', () => {
  render(
    <>
      <p id="test">label text</p>
      <Slider value={50} aria-labelledby="test" />
    </>
  );

  const slider = screen.getByRole('slider', { name: 'label text' });

  expect(slider).toBeVisible();
});
```

The reasoning being that this actually tests that the label is being applied in a manner that is similar to how a user would experience it.

For the second test a very similar:

```js
test('renders slider with aria-describedby', () => {
  render(
    <>
      <p id="test">descriptive text about the slider</p>
      <Slider value={50} aria-describedby="test" />
    </>
  );

  const slider = screen.getByRole('slider', { description: 'descriptive text about the slider' });

  expect(slider).toBeVisible();
});
```

Is what I would recommend, *but* the `description` option for `getByRole` is a new feature in RTL, so the version for that package would need to be bumped to incorporate it. I'll put up a bump PR for it in just a moment as that option is definitely something I'd like to leverage going forward.